### PR TITLE
[AppConfig] enable samples

### LINKED ISSUE #18541: [AppConfig] Enable running samples in pipeline

### PR DESCRIPTION
**Pull request by seankane-msft**
closes #18541

### CODE REVIEW

**Review by kristapratico — APPROVED** [review]

File: sdk/appconfiguration/tests.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ stages:
       AllocateResourceGroup: false
       ServiceDirectory: appconfiguration
       DeployArmTemplate: true
+      TestSamples: true
```
[the remark this answers is not in the document]
Comment (benbp) [reply sdk/appconfiguration/tests.yml:9]: Yes, that PR has been merged, so this needs to be changed.

[the remark this answers is not in the document]
Comment (benbp) [reply sdk/appconfiguration/tests.yml:9]:
```suggestion
      MatrixReplace:
        - TestSamples=.*/true
```

**Review by benbp — CHANGES_REQUESTED** [review]

**Review by benbp — APPROVED** [review]

**Pull request merged**